shell/shared/admin.js

```diff
@@ -759,6 +855,51 @@ if (Meteor.isServer) {
       }
 
       return { sent: true };
+    },
+    offerIpNetwork: function (token) {
+      if (!isAdmin()) {
+        if (tokenIsValid(token)) {
+          throw new Meteor.Error(403, "Offering IpNetwork is only allowed for logged in users " +
+            "(a token is not sufficient). Please sign in with an admin account")
+        } else {
+          throw new Meteor.Error(403, "User must be admin.");
+        }
+      }
+
+      var ipNetwork = new IpNetworkImpl(Meteor.userId());
+      var sturdyRef = waitPromise(ipNetwork.save({
+        sealFor: {
```

Comment:
Does this parameter actually make it through the RPC? I think that to deal with the quirks of node-capnp you need to do `ipNetwork.save({webkey: null})` here.